src/sql/workbench/browser/modelComponents/inputbox.component.ts

```diff
@@ -209,6 +222,9 @@ export default class InputBoxComponent extends ComponentBase implements ICompone
 		}
 
 		input.inputElement.required = this.required;
+
+		// save initial setting of aria label in case we change it later due to validation errors
+		this._defaultAriaLabel = this.ariaLabel;
```

Comment:
I looked around a bit more and it seems there is an input.setAriaLabel(this.ariaLabel); called on line ~207 which seems like sets input's aria label based on this aria label. We don't need to change that to use the new values... right?

Reply:
Re question 1--please refer to my earlier comment. I believe it answers this question.

I've tested this for both Windows and Mac, and although I'm new to screen reader support, the functionality appears to behave as expected.